microsoft-edge/webview2/roadmap.md

```diff
@@ -20,7 +20,7 @@ The WebView2 team is planning the following major efforts for future updates:
 
 * MacOS Preview
 * Xbox Preview
-* HoloLens Preview
+* HoloLens 2 Preview
```

Comment:
Remove HoloLens 2 Preview since this will now be available.

Reply:
done locally